examples/inspection/plot_permutation_importance.py

```diff
@@ -121,59 +130,109 @@
 #   therefore do not reflect the ability of feature to be useful to make
 #   predictions that generalize to the test set (when the model has enough
 #   capacity).
-ohe = rf.named_steps["preprocess"].named_transformers_["cat"]
-feature_names = ohe.get_feature_names_out(categorical_columns)
-feature_names = np.r_[feature_names, numerical_columns]
+#
+# The latter point explains why both the `random_num` and `random_cat` features
+# have a non-null importance. The former point explains why the `random_num`
+# has a really large importance in comparison with `random_cat` while we would
+# expect both random features to have a null importance.
```

Comment:
I think we can add these explanations to the points themselves rather than referencing them as "latter point" and "former point".

This way the reader does not need to scan back up to remember what the points were.